### PR TITLE
Fix task 1.1

### PR DESCRIPTION
**Pull request by EricaMatsinhe**

### CODE REVIEW

File: data/.gitkeep
Comment (Ndayao1) [inline data/.gitkeep:1]: LGTM

Comment (Ndayao1) [reply data/.gitkeep:1]: @EricaMatsinhe TLG

Comment (DrNyasha) [reply data/.gitkeep:1]: # 😄

**Review by Ndayao1 — APPROVED** [review]
@EricaMatsinhe TLGTM

**Pull request merged**